presto-native-execution/presto_cpp/main/SessionProperties.h

```diff
@@ -344,16 +348,34 @@ class SessionProperties {
   static constexpr const char* kUnnestSplitOutput =
       "native_unnest_split_output";
 
+  /// If this is true, then the protocol::SpatialJoinNode is converted to a
+  /// velox::core::NestedLoopJoinNode. Otherwise, it is converted to a
+  /// velox::core::SpatialJoinNode.
+  static constexpr const char* kMapProtocolSpatialJoinNodeToVeloxNLJNode =
+      "native_map_protocol_spatial_join_node_to_velox_nlj_node";
+
+  inline bool hasSessionPropertyVeloxConfigName(const std::string& key) {
+    auto sessionProperty = sessionProperties_.find(key);
+    return sessionProperty->second->getVeloxConfigName().has_value();
+  }
+
+  inline void updateSessionPropertyValue(const std::string& key, const std::string& value) {
+    auto sessionProperty = sessionProperties_.find(key);
+    sessionProperty->second->updateValue(value);
```

Comment:
Maybe better to add `VELOX_CHECK(sessionProperty != sessionProperties_.end())` here and in the previous function?